packages/kbn-alerts-ui-shared/src/alert_filter_controls/loading.tsx

```diff
@@ -12,7 +12,7 @@ import styled from '@emotion/styled';
 import { TEST_IDS } from './constants';
 
 const FilterGroupLoadingButton = styled(EuiButton)`
-  height: 34px;
+  height: 38px;
```

Comment:
I don't believe this is needed either. Instead, try setting a `size="s"` on line 20. This will set a small button size which seems to match the height of the input controls once they load.

Reply:
Thanks @ryankeairns .. Great observation. Fixed here : 8e885471230f8855fb11f5ae43b34a03231b67d9